src/models/document-content.ts

```diff
@@ -1,12 +1,12 @@
-import { types } from "mobx-state-tree";
-import { DocumentRowModel } from "./document-row";
+import { types, Instance } from "mobx-state-tree";
+import { DataSet } from "./data/data-set";
+import { ToolTileModel } from "./tools/tool-tile";
 
 export const DocumentContentModel = types
   .model("DocumentContent", {
-    // rows contain ToolTiles
-    rows: types.array(DocumentRowModel)
+    tiles: types.array(ToolTileModel),
     // data shared between tools
-    // shared: DataManager [TBD]
+    shared: types.maybe(DataSet)
   });
 
-export type DocumentContentModelType = typeof DocumentContentModel.Type;
+export type DocumentContentModelType = Instance<typeof DocumentContentModel>;
```

Comment:
I started the project using this method as the `.Type` was documented as the old method but I got type errors - it looks like some dependency update fixed that?

Reply:
I suppose so. We had been using the earlier convention with MST 1 and I saw the same note you did about that being the "legacy" syntax so I started changing it and it has seemed to work.